src/modules/keycloak-administration/domain/user.lock.ts

```diff
@@ -0,0 +1,15 @@
+export class UserLock<WasPersisted extends boolean> {
+    private constructor(
+        public person: Persisted<string, WasPersisted>,
```

Comment:
Use `PersonID` instead of string to signify that this is a person ID. (Also change it in `construct`)

Also, should this be using `Persisted<>`? The lock should always have a person ID, no matter if it has been saved or not. I don't see any reason for this class to use the generic at all right now

Reply:
done